pkgs/shells/lambda-mod-zsh-theme/default.nix

```diff
@@ -2,15 +2,15 @@
 
 let
   repo = "lambda-mod-zsh-theme";
-  rev = "eceee68cf46bba9f7f42887c2128b48e8861e31b";
+  rev = "c6445c79cbc73b85cc18871c216fb28ddc8b3d96";
 in stdenv.mkDerivation {
   name = "${repo}-${rev}";
```

Comment:
I would like to see rather our `lambda-mod-zsh-theme-unstable-year-month-day` notation rather then commit hashes as version number.